src/gluonts/transform/convert.py

```diff
@@ -759,3 +759,67 @@ def _empirical_cdf_inverse_transform(
             tensor_to_numpy(input_batch["intercepts"]),
         )
     return outputs
+
+
+class SparseToDense(FlatMapTransformation):
```

Comment:
The naming here may be misleading I think. The given array is dense, in the sense that it is represented as a dense array; however it is "sparse" in that it contains lots of zeros, so here we turn it into a compressed format.

I would personally like `DenseToSparse` better, but at this point it's clear that this creates confusion. Maybe `ToIntervalSizeFormat` is more to the point?

Reply:
`ToIntervalSizeFormat` it is. Thanks.